server/sonar-server/src/main/java/org/sonar/server/user/index/UserIndex.java

```diff
@@ -137,7 +139,7 @@ public UserDoc getNullableByLogin(String login) {
     }
 
     request.setQuery(QueryBuilders.filteredQuery(query,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "filteredQuery"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)